assembly/pom.xml

```diff
@@ -89,4 +78,41 @@
     </plugins>
   </build>
 
+  <profiles>
+    <profile>
+      <id>targz</id>
+      <properties>
+        <assembly.format>tar.gz</assembly.format>
+      </properties>
+    </profile>
+
+    <profile>
+      <id>scala-2.10</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
```

Comment:
Do not use `activeByDefault`. It does not do what you think it does.

If you want to choose between 2.10 and 2.11 using profiles, you need to do what Spark does and condition it on a property. (`-Dscala-2.10` in the case of Spark, since 2.11 is the default there)